src/Select/Select.story.tsx

```diff
@@ -571,3 +571,30 @@ export const WithCustomProps = () => {
 UsingRefToControlFocus.story = {
   name: "using ref to control focus",
 };
+
+export const PasteCsvValueInSelect = (props) => {
+  const [state, setState] = React.useState([]);
+
+  const handleChange = (value) => {
+    setState(value);
+  };
+
+  return (
+    <Select
+      defaultValue={[partnerCompanyName[0].value, partnerCompanyName[2].value]}
+      noOptionsMessage={() => "No options"}
+      placeholder="Please select inventory status"
+      options={PCNList}
+      labelText="Select PCN"
+      className="Select"
```

Comment:
Do we need this className?

Reply:
className is not required, but added because other stories have it.